docs/source/guides/contributing.rst

```diff
@@ -24,34 +34,22 @@ Clone the `ModdingDocs repo <https://github.com/R2Northstar/ModdingDocs/>`_, e.g
 
 Opened the clone repo.
 
-Setup a `Python virtual environment <https://docs.python.org/3/tutorial/venv.html>`_
-(this is not strictly necessary but can help keep your Python install clean)
+(on Windows, replace `python3` with `py`)
 
-.. code:: bash
 
-    # Create virtual environment (the second `venv` is the name of the folder of the virtual environment)
-    python3 -m venv venv
-
-    # On Windows, activate with:
-    venv\Scripts\activate.bat
+.. code:: bash
 
-    # On Linux, activate with:
-    source venv/bin/activate
+    # Install Poetry, the build tool used by ModdingDocs
+    python3 -m pip install poetry
```

Comment:
I'm assuming I can still slap that in a `venv` or does `poetry` create `venv` for you? Just so I can avoid an [XKCD 1987](https://xkcd.com/1987/) scenario on my own machine :P

Reply:
Poetry creates a venv for you, and completely manages the venv.